src/utils/component.js

```diff
@@ -39,4 +44,173 @@ const traverseFragment = (component) => {
   return fragmentComponents;
 };
 
-export { traverseFragment, findReactRootContainer };
+const colorize = (text, styles) => {
+  return [`%c${text}`, styles];
+};
+
+const findReactRootNode = (element) => {
+  if (!element) return null;
+
+  if (
+    Object.keys(element).some(
+      (key) =>
+        key.startsWith('__reactFiber$') ||
+        key.startsWith('__reactInternalInstance$'),
+    )
+  ) {
+    return element;
+  }
+
+  const childrenArray = Array.from(element.children);
+  let reactNode = null;
+
+  childrenArray.some((children) => {
+    reactNode = findReactRootNode(children);
+
+    return reactNode;
+  });
+
+  return reactNode;
+};
+
+const findReactFiber = (element) => {
+  const reactKey = Object.keys(element).find(
+    (key) =>
+      key.startsWith('__reactFiber$') ||
+      key.startsWith('__reactInternalInstance$'),
+  );
```

Comment:
[findReactRootNode 함수에서도 비슷한 방식](https://github.com/Team-macoss/con.chat/blob/66e67ce294d4d02932d1664f45971c6eaf06c3c6/src/utils/component.js#L55-L62)으로 키를 찾는 로직이 있는 것 같습니다!
이 부분을 하나의 유틸 함수로 합쳐 중복을 제거해도 좋을 것 같습니다~

```suggestion
 const findReactKey = (element, keys) => {
  return keys.find(
    (key) =>
      key.startsWith('__reactFiber$') ||
      key.startsWith('__reactInternalInstance$')
  );
};
```

Reply:
트리 순회 알고리즘은 추후 수정될 여지가 있어서 유틸화를 따로 진행하지는 않았습니다. 제안주신 사항 리팩토링 테스크에서 고려해보도록 하겠습니다. 감사합니다!